docs/react-v9/contributing/dev-env.md

```diff
@@ -24,8 +24,8 @@ This document describes how to set up your development environment and contribut
 
 Open a command line and run:
 
-- `node -v`: Should be `18.x.x` Newer versions may not work.
-- `npm -v`: Should be >= 6. If not, run `npm install -g npm`.
+- `node -v`: Should be `^20.x.x`.
+- `npm -v`: Should be >= 10. If not, run `npm install -g npm`.
```

Comment:
not related: but do we care about NPM? (we use `yarn`)

Reply:
Good question. Tbh, I think not really for a project, but for the sake of having more recent version of `npm` on the machine in general :)

Reply:
we don't , I pointed this out during PR review of this docs but It might got lost

Reply:
Thanks guys, updated 👍🏻